otcextensions/osclient/cce/v2/cluster_node.py

```diff
@@ -131,9 +131,12 @@ def take_action(self, parsed_args):
             cluster = client.find_cluster(parsed_args.cluster,
                                           ignore_missing=False)
             for node in parsed_args.node:
+                obj = client.find_cluster_node(
```

Comment:
A new request means a new API mock is required in unit tests. Due to that currently tests are failing